### PR TITLE
Fix proxies docs.

### PR DESCRIPTION
**Pull request by ehuss**
There were a few minor mistakes in #2544, most notably that the file wasn't added to SUMMARY.md which means it doesn't render at all.

### CODE REVIEW

**Review by kinnison — APPROVED** [review]
Thanks for this.

I wonder if it'd be possible to render the docs in a PR and somehow view them?  Gitlab lets you view artifacts for a CI pipelie, is there an equivalent in github actions?

**Comment by ehuss** [comment]
There looks like there is an [upload-artifact action](https://docs.github.com/en/free-pro-team@latest/actions/guides/storing-workflow-data-as-artifacts), but it requires a REST API to fetch the artifacts.  I have never used it, so I am not too familiar with it.

BTW, I'm still planning to implement the ability to view the "master" docs (after PRs merge to master), but I think I might wait till after the release, since that will make it easier once the `stable` branch has been updated.

**Pull request merged**
